docs/reference-guides/monitoring-v2-configuration/receivers.md

```diff
@@ -211,6 +213,87 @@ url http://rancher-alerting-drivers-sachet.ns-1.svc:9876/alert
 
 <!-- https://github.com/messagebird/sachet -->
 
+## Telegram
+
+### Enabling the Telegram Receiver for Rancher Managed Clusters
+
+The Telegram receiver is not a native receiver and must be enabled before it can be used. You can enable the Telegram receiver for a Rancher managed cluster by going to the Apps page and installing the rancher-alerting-drivers app with the Telegram option selected.
+
+1. In the upper left corner, click **☰ > Cluster Management**.
+1. On the **Clusters** page, go to the cluster where you want to install `rancher-alerting-drivers` and click **Explore**.
+1. In the left navigation bar, click on **Apps**.
+1. Click the **Alerting Drivers** app.
+1. Click on **Install**.
+1. In the page that opens next, make sure that **Enable SMS** checkbox is selected.
+1. Take note of the namespace used as it will be required in a later step.
+
+### Configure a PrometheusRule
+
+To test our Telegram setup, let's create a **PrometheusRule** that will fire all the time. 
+
+:::caution NOTE
+This rule is suggested only for the purpose of testing if Telegram alerts work as expected. Do not leave it on beyond the purpose of testing.
+:::
+
+1. Click on **Monitoring**.
+2. Click on **Advanced**.
+3. Click on **PrometheusRules** and click on **Create** to create a new rule.
```

Comment:
```suggestion
1. Click on **PrometheusRules > Create**.
```
Docusaurus knows to render the numbering correctly, in case we move steps around or add/remove instructions

Reply:
Yep, I'm aware. But the IDE numbered them 1, 2, 3, etc. And, AFAIK, it's a markdown thing, not a Docusaurus thing. E.g.:
1. one
1. two
1. three